### PR TITLE
Martini - implement setter for Logger

### PR DESCRIPTION
**Pull request by ovr**
Hey!

I think this will be rly cool
Via I am customizing martini classic

image

Thanks

### CODE REVIEW

**Comment by Kistamushken** [comment]
:+1:

File: martini.go [hunk 1]
```diff
@@ -57,6 +57,12 @@ func (m *Martini) Action(handler Handler) {
 	m.action = handler
 }
 
+// Action sets the logger
```
Comment (mvader) [inline martini.go]: Should be `Logger sets the logger`. Anyway, I'd name the method `SetLogger`. By the name, `Logger` suggests that it's a getter.

Comment (ovr) [inline martini.go]: @mvader 
https://github.com/ovr/martini/blob/70f17d289f9ca20ff7beed9ab501b163b0cd5e42/martini.go#L55
So I named method like 'Logger' via I saw `Action` method (it is another setter)

Comment (mvader) [inline martini.go]: Ok, fair enough, just replace the `Action` with `Logger` in the comment and LGTM

**Comment by ovr** [comment]
@mvader 

> Ok, fair enough, just replace the Action with Logger in the comment and LGTM

Done, Just wait CI

**Comment by mvader** [comment]
Merging, thanks for the contribution :+1:

**Comment by ovr** [comment]
@mvader 

Thank you too!
Take this :cake:

**Pull request merged**